### PR TITLE
ci: use Go 1.10/1.11, test with -race

### PR DESCRIPTION
**Pull request by adamdecaf**
Noticed when looking over this project for https://github.com/gofrs/help-requests/issues/26

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ clone_folder: 'c:\gopath\src\github.com\theckman\go-flock'
 
 environment:
   GOPATH: 'c:\gopath'
-  GOVERSION: '1.9.2'
+  GOVERSION: '1.11.0'
```
Comment (jadr2ddude) [inline appveyor.yml]: appveyor cannot get this, possibly because there is no go 1.11.0, only 1.11

**Comment by theckman** [comment]
Let's work on this one after we migrate, since we'll need to touch these files anyhow.

**Comment by theckman** [comment]
Cycling the PR to try and trigger fresh PR runs.

**Comment by adamdecaf** [comment]
@theckman Done.

**Pull request merged**